packages/liveblocks-react-comments/src/primitives/Comment/index.tsx

```diff
@@ -117,5 +144,16 @@ if (process.env.NODE_ENV !== "production") {
   CommentMention.displayName = COMMENT_MENTION_NAME;
 }
 
+function toAbsoluteURL(url: string): string | undefined {
+  // Check if the URL already contains a scheme
+  if (url.startsWith("http://") || url.startsWith("https://")) {
+    return url;
+  } else if (url.startsWith("www.")) {
+    // If the URL starts with "www.", prepend "https://"
+    return "https://" + url;
+  }
+  return;
+}
+
```

Comment:
We can move this to `/src/primitives/Comment/utils.ts`.

Reply:
Done! 

I am not completely certain how I feel about this method. If our customers are using the high level `Comment` component, then it works all right, but if customers intend to customize the `Link` component via the `renderLink` prop, we are expecting them to build their own implementation of this method if they want to ensure the links are absolute, which should be most of the time. 
An alternative could be providing both absolute url and text in the comment body but this will interfere with our upcoming potential design for custom links which will have the same two properties. What do you think @marcbouchenoire?

Reply:
I tweaked the way both `renderLink` props work so that instead of `url` we provide `href` and `text`: we convert the URL to absolute and pass it as `href` so that users don't have to handle it, and for auto-links we provide the original URL as `text` (which means that the users' implementations won't need to change when custom links arrive later on: `<a href={href}>{text}</a>` will remain valid).

Reply:
(`text` is now `children`: https://github.com/liveblocks/liveblocks/pull/1131#discussion_r1324317386)